extensions/test-extension/src/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { id } from './id';
+import init from './init';
+
+/**
+ *
+ */
+const testExtension = {
+  /**
+   * Only required property. Should be a unique value across all extensions.
+   */
+  id,
+  preRegistration({
+    servicesManager,
+    commandsManager,
+    extensionManager,
+    configuration = {},
+  }) {
+    init({ servicesManager, commandsManager, extensionManager, configuration });
```

Comment:
Please import as "preRegistration" and include the function directly, rather than adding an extra hop here.  That avoids the extra communication about parameters between index.tsx and init